.azure/gpu-integrations.yml

```diff
@@ -0,0 +1,95 @@
+# Python package
+# Create and test a Python package on multiple Python versions.
```

Comment:
Maybe we may mention integration here as well?

Reply:
hmm it was some old unrelated comment so updated this and the other one too